tests/jq.test

```diff
@@ -1352,6 +1352,10 @@ unique
 []
 [null,null,null,null]
 
+min(1,2,0.1)
+null
+0.1
+
```

Comment:
Add tests with max, and also with `empty` as its argument.

Reply:
Done.